nipype/interfaces/tests/test_base.py

```diff
@@ -545,7 +511,8 @@ class DerivedInterface1(nib.BaseInterface):
         input_spec = InputSpec
         _version = '0.10'
     obj = DerivedInterface1()
-    yield assert_not_raises, obj._check_version_requirements, obj.inputs
+    #NOTE_dj: removed yield assert_not_raises
+    obj._check_version_requirements(obj.inputs)
```

Comment:
yes, if `obj._check_version_requirements(obj.inputs)` raises any exception the test will fail, so  `assert_not_raises` is not needed IMO